pilot/pkg/model/envoyfilter.go

```diff
@@ -203,13 +203,7 @@ func (efw *EnvoyFilterWrapper) KeysApplyingTo(applyTo ...networking.EnvoyFilter_
 }
 
 func genFullName(namespace, name string) string {
```

Comment:
I think it is better to remove the function and concatenate directly when we intialize EnvoyFilterConfigPatchWrapper

Reply:
updated